backend/sozisel/lib/sozisel/model/event_results.ex

```diff
@@ -29,4 +30,31 @@ defmodule Sozisel.Model.EventResults do
   def change_event_result(%EventResult{} = event_result, attrs \\ %{}) do
     EventResult.create_changeset(event_result, attrs)
   end
+
+  def get_all_event_results(%LaunchedEvent{id: launched_event_id}) do
+    from(res in EventResult, where: res.launched_event_id == ^launched_event_id)
+    |> Repo.all()
+  end
+
+  def quiz_result_summary(%LaunchedEvent{} = event_launched) do
+    quiz_results = get_all_event_results(event_launched)
+    # IO.inspect(quiz_results)
+
+    Enum.each(quiz_results, fn event_data -> 
+      %{result_data: result_data} = Utils.from_deep_struct(event_data) 
+      |> Map.take([:result_data])
+
+      %{quiz_time: quiz_time} = 
+      result_data
+      |> Map.take([:quiz_time])
```

Comment:
`result_data.quiz_time` looks the same
And you know that `Enum.each` does not return anything?

Reply:
Yes, I know, but I tried my best to parse this data and I pushed the branch that you could just see it and help me.